FHEM/14_SD_Keeloq.pm

```diff
@@ -562,7 +564,7 @@ sub Set($$$@) {
 
 				### Zusammenführen
 				my $bits = reverse (sprintf("%032b", $encoded)).reverse($models{$model}{Channel}{$channel}).reverse($Serial_send).reverse($buttonbits).reverse($bit64to71);
-				$Repeats = 20 if ($cmd eq "shade");			# special, command shade = 20 repeats
+				$Repeats = 15 if ($cmd eq "shade");			# special, command shade = 20 repeats = 2,34 s / 15 = 1,75s / userreport: 12 repeats ok
```

Comment:
Erklärt das eventuell #632 ?

Reply:
So richtig kann ich mir das nicht erklären.
Die Repeats 15 

https://github.com/RFD-FHEM/RFFHEM/blob/4c104970774d1fad50865367fb51f841f2c2d749/FHEM/14_SD_Keeloq.pm#L567

sind nur für das shade Kommando.

Wieso bei dem User das auf 2 Kommandos passiert muss mit Ihm genauer hinterfragt werden. Die Konstellation im Modul läuft bisher bei allen so außer dem "Fehlverhalten" von @Banabas.

Reply:
Wäre das ein Hinweis in de Commandref wert?

Reply:
Ich hätte es zum Attribut repeats gepackt.

Reply:
Das Attribut Repeats hat nichts mit DEN Repeats 15 zu tun. Die "normalen" Repeats fürs senden sind im Attribut Repeats verankert. Aus dem Grunde, das die 20 fest in dem Bastenbuden... Projekt verankert war, so wurde dies übernommen. Wo anders fand ich auch mal, das das eine Kommando eben mehrere Wiederholungen benötigt, deswegen wurde es vom normalen Attribut abgekapselt.